.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Files**
+
+Attach or link to .glb files or avatar URL that trigger the bug. 
+
+In addition, make sure to run those files through the Ready Player Me Unity SDK example scenes first. If you encounter errors or warnings, try to make sure they are not responsible for the issue and file a bug report with the software that generated the glTF file as well.
```

Comment:
Do we need to point our the full name `Ready Player Me Unity SDK` or `Our Unity SDK` would be fine?